CHANGELOG/CHANGELOG-1.20.md

```diff
@@ -2665,7 +2665,7 @@ filename | sha512 hash
   When we switched to go-runner for building the apiserver,
   controller-manager, and scheduler server components, we no longer
   reference the individual architectures in the image names, specifically
-  in the 'FROM' directive of the server image Dockerfiles.
+  in the 'FROM' directive of the server image Dockerfile.
```

Comment:
same as above, i.e. `Dockerfiles`

Reply:
Hi, @wilsonehusin, all abolve have been updated already, looking for your review with pleasure.